integration_tests/integration/serviceProviderReferrals.spec.js

```diff
@@ -241,10 +242,20 @@ describe('Service provider referrals dashboard', () => {
         description: 'Service User is helped to secure a tenancy in the private rented sector (PRS)',
       },
     ]
+
     const serviceCategory = serviceCategoryFactory.build({ name: 'accommodation', desiredOutcomes })
+    const accommodationIntervention = interventionFactory.build({
+      contractType: { code: 'SOC', name: 'Social inclusion' },
+      serviceCategories: [serviceCategory],
+    })
+
     const selectedDesiredOutcomesIds = [desiredOutcomes[0].id, desiredOutcomes[1].id]
     const referralParams = {
-      referral: { serviceCategoryId: serviceCategory.id, desiredOutcomesIds: selectedDesiredOutcomesIds },
+      referral: {
+        interventionId: accommodationIntervention.id,
+        serviceCategoryId: serviceCategory.id,
```

Comment:
should we replace this with (or just add) 'serviceCategoryIds' here now too?

Reply:
It'll break if we remove this, as it's still used on the action plan page, but I've added `serviceCategoryIds` and added a TODO for removing the old field when that task is picked up.